ace/validator.go

```diff
@@ -328,9 +328,9 @@ func validatePodMetadata(metadataURL string, pm *schema.PodManifest) results {
 func validateAppAnnotations(metadataURL string, pm *schema.PodManifest, app *schema.RuntimeApp, img *schema.ImageManifest) results {
 	r := results{}
 
-	// build a map of expected annotations by merging app.Annotations
+	// build a map of expected annotations by merging img.Annotations
 	// with PodManifest overrides
-	expectedAnnots := app.Annotations
+	expectedAnnots := img.Annotations
 	a := pm.Apps.Get(app.Name)
```

Comment:
Great catch, although this raises another point of confusion for me - isn't this line just the same as app??